concent_api/core/utils.py

```diff
@@ -176,3 +178,25 @@ def adjust_format_name(output_format: str) -> str:
     if output_format.upper() == 'JPG':
         return 'JPEG'
     return output_format.upper()
+
+
+class BlocksHelper:
+    def __init__(self, sci: SmartContractsInterface) -> None:
+        self._sci = sci
+
+    def get_latest_existing_block_at(self, timestamp: int) -> Block:
```

Comment:
That `at` at the end of this name is confusing. Can't this function be called `get_latest_existing_block()`?

Reply:
@Jakub89 maybe `get_latest_existing_block_since`?

Reply:
@Jakub89 @kbeker I am fine with `get_latest_existing_block_at`